### PR TITLE
Fixes on randomize methods in Persons[SORDEV-8466] and Event[SORDEV-5…

### PR DESCRIPTION
**Pull request by mk-sgent**

### CODE REVIEW

**Comment by mk-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/64/allure/

**Comment by mk-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/68/allure/

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature [hunk 1]
```diff
@@ -195,54 +195,6 @@ Feature: Create events
...
-    And I apply on the APPLY FILTERS button from Event
-    And I check that number of displayed Event results is 0
-    And I click on the RESET FILTERS button from Event
-    Then I select Signal filter from quick filter
-    And I select Event filter from quick filter
-    And I select Screening filter from quick filter
-    And I select Cluster filter from quick filter
-    And I select Dropped filter from quick filter
-    And I click on the RESET FILTERS button from Event
-
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature:245]: why was this test removed?

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java [hunk 2]
```diff
@@ -191,32 +207,44 @@ public PersonDirectorySteps(
...
-          webDriverHelpers.selectFromCombobox(BIRTH_DAY_COMBOBOX, dayOfBirth.toString());
+          webDriverHelpers.selectFromCombobox(
+              BIRTH_DAY_COMBOBOX,
+              getRandomNumberForBirthDateDifferentThanCreated(
+                      apiState.getLastCreatedPerson().getBirthdateDD(), 1, 27)
+                  .toString());
         });
 
     Then(
         "I change present condition filter to random for Person",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java:238]: let's update this then to reflect that we're using the person created via api

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature (hunk 1 above)
Comment (mk-sgent) [reply sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature:245]: This test is moved to EventFilters.feature

**Comment by mk-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/69/allure/

File: sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature (hunk 1 above)
Comment (rdutu-vg) [reply sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature:245]: but I can't see the change here on this PR

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java [hunk 3]
```diff
@@ -47,6 +62,7 @@
 public class PersonDirectorySteps implements En {
   private final WebDriverHelpers webDriverHelpers;
   protected Person createdPerson;
+  Faker faker = new Faker();
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java:65]: inject it, don't create it by using "new"

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

**Pull request closed without merging**